decidim-initiatives/README.md

```diff
@@ -44,7 +44,7 @@ CREATE EXTENSION pg_trgm;
 
 ## Deactivating authorization requirement and other module settings
 
-Some of the settings of the module need to be set in the code of your app, for example in the file `config/initializers/decidim.rb`.
+Some of the settings of the module need to be set in the code of your app using the [environment variables](https://docs.decidim.org/en/develop/configure/environment_variables)
```

Comment:
```suggestion
After that, configure your application using the environment variables as presented in the [documentation](https://docs.decidim.org/en/develop/configure/environment_variables.html).
```

Reply:
Can you fix the instructions after this line? Until line 71 (`### Initiatives signatures`) we should only refer to the env vars (and then we should present the initializer)

Reply:
done in 68f3d10ba1c541967fed066e1839ce49b64de25c